src/PrestaShopBundle/Entity/Repository/StockManagementRepository.php

```diff
@@ -380,4 +396,131 @@ protected function bindMaxResultsValue(Statement $statement, QueryParamsCollecti
             PDO::PARAM_INT
         );
     }
+
+    /**
+     * Store the number of rows found in a previous query executed with SQL_CALC_FOUND_ROWS
+     */
+    protected function getFoundRows()
+    {
+        $statement = $this->connection->prepare('SELECT FOUND_ROWS()');
+        $statement->execute();
+        $rowCount = (int)$statement->fetchColumn();
+        $statement->closeCursor();
+
+        return $rowCount;
+    }
+
+    /**
+     * @param array $row
+     */
+    protected function addCombinationName(array &$row)
+    {
+        $query = 'SELECT GROUP_CONCAT(
+                        DISTINCT CONCAT(agl.name, " - ", al.name)
+                        SEPARATOR ", "
+                    )
+                    FROM '.$this->tablePrefix.'product_attribute pa
+                    LEFT JOIN '.$this->tablePrefix.'product_attribute_combination pac ON (
+                        pac.id_product_attribute = pa.id_product_attribute
+                    )                    
+                    LEFT JOIN '.$this->tablePrefix.'attribute a ON (
+                        a.id_attribute = pac.id_attribute
+                    )
+                    LEFT JOIN '.$this->tablePrefix.'attribute_lang al ON (
+                        a.id_attribute = al.id_attribute
+                        AND al.id_lang = :language_id
+                        AND LENGTH(TRIM(al.name)) > 0
+                    )
+                    LEFT JOIN '.$this->tablePrefix.'attribute_group ag ON (
+                        ag.id_attribute_group = a.id_attribute_group
+                    )
+                    LEFT JOIN '.$this->tablePrefix.'attribute_group_lang agl ON (
+                        ag.id_attribute_group = agl.id_attribute_group
+                        AND agl.id_lang = :language_id
+                        AND LENGTH(TRIM(agl.name)) > 0
+                    )                    
+                    WHERE pa.id_product=:id_product AND pa.id_product_attribute=:id_product_attribute
+                    ';
+        $statement = $this->connection->prepare($query);
+        $statement->bindValue('id_product', (int)$row['product_id'], \PDO::PARAM_INT);
+        $statement->bindValue('id_product_attribute', (int)$row['combination_id'], \PDO::PARAM_INT);
+        $statement->bindValue('language_id', (int)$this->languageId, \PDO::PARAM_INT);
+        $statement->execute();
+        $row['combination_name'] = $statement->fetchColumn(0);
+        $statement->closeCursor();
+    }
+
+    /**
+     * @param array $row
+     */
+    protected function addProductFeatures(array &$row)
```

Comment:
same here, if it's intended and not qn issue : ignore my previous comment about it ;)